examples/component-provider/main.go

```diff
@@ -18,23 +18,33 @@ package main
 
 import (
 	"context"
+	"fmt"
+	"os"
 
+	p "github.com/pulumi/pulumi-go-provider"
 	"github.com/pulumi/pulumi-go-provider/examples/component-provider/nested"
 	"github.com/pulumi/pulumi-go-provider/infer"
 )
 
 func main() {
-	p, err := infer.NewProviderBuilder().
-		WithNamespace("example-namespace").
```

Comment:
@mikhailshilkov should these examples call `WithNamespace` and what would a good value be?

Reply:
I think it's a good practice, so that generated SDKs aren't namespaced to default `pulumi`. You don't like `example-namespace`?

Reply:
I don't think it very descriptive because the concept of namespaces lacks definition.  One might mistakenly think a namespace is at the provider level, to contain the provider's types. I believe it is at the org-level, to contain an organization's providers. 

How about `WithNamespace("examples")` since these examples are in an `examples/` folder.